### ISSUE TITLE
losses property in layer class is empty

### ISSUE THREAD

**Issue by zy871125746** [keras-team/keras#10032]
I am trying to visualize the loss of each layer in CNN by tensor-board with Keras to see how the losses changes in every layer timely. The current problem is i can not get loss for each layer. I have seen there is a property name losses from Model.layers[].losses but it is empty.

Here is part of my code which is modified from the TensorBoard class inside callback module in keras:

     def set_model(self, model):
            self.model = model
            if K.backend() == 'tensorflow':
                self.sess = K.get_session()
            if self.histogram_freq and self.merged is None:
                for layer in self.model.layers:
                    layer_losses = layer.losses
                    layer_loss_name = layer.name+"/loss"
                    tf.summary.histogram(layer_loss_name,layer_losses)
